MAUI/DataGrid/pull-to-refresh.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: PullToRefresh in .NET MAUI DataGrid control | Syncfusion
```

Comment:
title: Pull To Refresh in .NET MAUI DataGrid Control | Syncfusion

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/f18ad841-300a-4c93-8bf4-0a6c95fbb1ce)
Had updated as discussed.